ui/src/app/store/pod/types.ts

```diff
@@ -55,6 +55,13 @@ export type Pod = Model & {
   zone: number;
 };
 
```

Comment:
A comment here to explain that the server sends both `Pod` and `PodDetails` as an optimisation for details pages might make this clearer.

Reply:
Done